flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -458,29 +460,46 @@ public enum JobStoreType {
                     .withDescription(
                             Description.builder()
                                     .text(
-                                            "Determines which scheduler implementation is used to schedule tasks. Accepted values are:")
-                                    .list(
-                                            text("'Default': Default scheduler"),
-                                            text(
-                                                    "'Adaptive': Adaptive scheduler. More details can be found %s.",
-                                                    link(
-                                                            "{{.Site.BaseURL}}{{.Site.LanguagePrefix}}/docs/deployment/elastic_scaling#adaptive-scheduler",
-                                                            "here")),
-                                            text(
-                                                    "'AdaptiveBatch': Adaptive batch scheduler. More details can be found %s.",
-                                                    link(
-                                                            "{{.Site.BaseURL}}{{.Site.LanguagePrefix}}/docs/deployment/elastic_scaling#adaptive-batch-scheduler",
-                                                            "here")))
+                                            "Determines which scheduler implementation is used to schedule tasks. "
+                                                    + "If this option is not explicitly set, batch jobs will use the "
+                                                    + "'AdaptiveBatch' scheduler as the default, while streaming jobs "
+                                                    + "will default to the 'Default' scheduler. ")
                                     .build());
 
     /** Type of scheduler implementation. */
-    public enum SchedulerType {
+    public enum SchedulerType implements DescribedEnum {
         /** @deprecated Use {@link SchedulerType#Default} instead. */
         @Deprecated
-        Ng,
-        Default,
-        Adaptive,
-        AdaptiveBatch
+        Ng(),
```

Comment:
It's better to describe it as "Deprecated. Use Default scheduler instead."

Then we can remove the paramless ctor.